bindings/haskell/Cargo.toml

```diff
@@ -17,19 +17,12 @@
 
 [package]
 name = "opendal-hs"
-publish = false
-
-authors.workspace = true
-edition.workspace = true
-homepage.workspace = true
-license.workspace = true
```

Comment:
Those values are required, please only set `version` by hand.

Reply:
changed in new commit